src/scripts/ordo_subsets.py

```diff
@@ -18,19 +18,9 @@
 }
 
 
-def create_ordo_subsets_robot_template(
-    class_subsets_tsv_path: str, mondo_mappings_path: str, outpath: str,
-):
-    """Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
-    # Read: mondo.sssom.tsv
-    mondo_df = pd.read_csv(mondo_mappings_path, sep='\t', comment='#').rename(columns={
-        'subject_id': 'mondo_id',
-        'subject_label': 'mondo_label',
-    })
-    # - keep only exact matches
-    mondo_df = mondo_df[mondo_df['predicate_id'] == 'skos:exactMatch']
-
-    # Read: ORDO class subsets query results
+def get_formatted_subsets_df(class_subsets_tsv_path: str):
```

Comment:
## `get_formatted_subsets_df()`
In `ordo_subsets.py`, which is part of #510, there existed some code I needed to re-use; namely, the reading and formatting of `tmp/ordo-subsets.tsv`. So I created this function for that, which is now used in 2 places. That's what the whole diff in `ordo_subsets.py` is about, just moving some code into this new function.